introduction.md

```diff
@@ -5,3 +5,14 @@ I am the man they call borozoro
 My favorite Code School path is the JavaScript path.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?